File: samples/conference/payments-bc/src/main/java/com/microsoft/conference/payments/domain/Events/PaymentRejected.cs
using System;
using ENode.Eventing;

namespace Payments
{
    public class PaymentRejected : DomainEvent<Guid>
    {
        public Guid OrderId { get; private set; }
        public Guid ConferenceId { get; private set; }

        public PaymentRejected() { }
        public PaymentRejected(Guid orderId, Guid conferenceId)
        {
            OrderId = orderId;
            ConferenceId = conferenceId;
        }
    }
}

File: samples/conference/payments-bc/src/main/java/com/microsoft/conference/payments/domain/events/PaymentRejected.java
package com.microsoft.conference.payments.domain.events;

import org.enodeframework.eventing.DomainEvent;

public class PaymentRejected extends DomainEvent<String> {
    public String orderId;
    public String conferenceId;

    public PaymentRejected() {
    }

    public PaymentRejected(String orderId, String conferenceId) {
        this.orderId = orderId;
        this.conferenceId = conferenceId;
    }
}
